core/python/ntuplewriter.py

```diff
@@ -658,6 +654,9 @@ def add_fatjets_subjets(process, fatjets_name, groomed_jets_name, jetcorr_label=
                  elSource=cms.InputTag('slimmedElectrons'),
                  getJetMCFlavour=(not useData)
                  )
+#manually override parton & genjet matching even though we set getJetMCFlavour false...
+#ToDo: figure out more beautiful solution
```

Comment:
Hmm I'm having a look to see if this as been solved - someone has already mentioned it: https://github.com/cms-sw/cmssw/issues/20057